src/components/Footer.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
```

Comment:
All the comments I mentioned above about the order of the main component and styled components besides the correct way to declare a functional component should be applied in this file.

Reply:
Done

Reply:
Ok I see you misunderstood me.

The correct way I meant is this.
`function Footer() {
  return (
    <AppFooter>
      <footer>
        <h4>
          Created by
          <A href="https://github.com/WinterFirebird" target="_blank">Arman Grigoryan</A>
        </h4>
      </footer>
    </AppFooter>
  );
}

function Footer() {
  return (
    <AppFooter>
      <footer>
        <h4>
          Created by
          <A href="https://github.com/WinterFirebird" target="_blank">Arman Grigoryan</A>
        </h4>
      </footer>
    </AppFooter>
  );
}

`